### PR TITLE
update service load balancer test to gracefully roll out

### PR DESCRIPTION
**Pull request by deads2k**
This adds a readyz=false state into the service being served by the load balancer to allow time for the load balancer to realize that an endpoint is no longer ready.  During this time period, requests are still handled by the server, so we should see a reduction in the disruption as a result.

### CODE REVIEW

**Comment by deads2k** [comment]
```
                "message": "Back-off pulling image \"quay.io/openshift/community-e2e-images:e2e-1-k8s-gcr-io-e2e-test-images-agnhost-2-39-D3QKX7Bo93z7tlZq\"",
                "reason": "ImagePullBackOff"
```

File: test/extended/util/image/image.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func init() {
 
 		// allowed upstream kube images - index and value must match upstream or
 		// tests will fail (vendor/k8s.io/kubernetes/test/utils/image/manifest.go)
-		"k8s.gcr.io/e2e-test-images/agnhost:2.33":   1,
+		"k8s.gcr.io/e2e-test-images/agnhost:2.39":   1,
```
Comment (yselkowitz) [inline test/extended/util/image/image.go:68]: This version is defined by the kubernetes test suite, and is being updated to 2.36 as part of the k8s 1.24.0 rebase in #27181.

Comment (yselkowitz) [reply test/extended/util/image/image.go:68]: Also, all the rest of the tests in origin that use `agnhost` also need to be updated to the same version simultaneously.

**Comment by deads2k** [comment]
This is working in the sense that we go ready=false now.  We still have an outage, but it's an improvement.  To be continued once the main o/o update is done.

**Comment by deads2k** [comment]
/hold cancel

this is now ready to merge.

**Pull request merged**